CONTRIBUTING.md

```diff
@@ -1,4 +1,44 @@
 # Contributing
+## App Architecture
+This section describes how the application is structured, mainly the Python backend. This does not attempt to cover any information about the React frontend. As of this writing the app is currently under a major refactoring to make the application more robust. There is code that uses the deprecated [RequestParser](https://flask-restful.readthedocs.io/en/latest/reqparse.html) which is being removed from the application in favor of [Marshmallow](https://marshmallow.readthedocs.io/en/stable/). It is expected that all new functionality uses Marshmallow.
+
+This application uses the [Flask](https://palletsprojects.com/p/flask/) micro-framework backed by an [sqlLite](https://sqlite.org/index.html) database ([Postgres](https://www.postgresql.org/) for production) utilizing [SQLAlchemy](https://docs.sqlalchemy.org/en/14/) as the ORM. [Flask-Restful](https://flask-restful.readthedocs.io/en/latest/) is used for routing to encourage Restful routes/resources. [Marshmallow](https://marshmallow.readthedocs.io/en/stable/) is being used for input validation, serialization, and deserialization. Flask-mail is currently being used to send mail, and Jinja is used for templating the email messages. However, the mail library has been deprecated and there is an open issue to switch to another library. Flake8 is currently installed for linting, but not really being used. However, we will most likely be using [black](https://github.com/psf/black) in the future or a combination of Flake8 for linting and black for formatting. For database migrations we recently installed alembic and will be using that in the future. For now, while the app is under development we drop the db and rebuild whenever there is a change. There are three different environments the application is setup to run in. Development, testing, and production environments. For testing we are using Pytest. It is an expectation that all new functionality, changes in behavior, or bug fixes will be covered under automated tests.
```

Comment:
For now, while the app is under development,
(add a comma after development)

Change "There are three different environments the application is setup to run in. Development, testing, and production environments." to "There are three different environments the application is setup to run in: development, testing, and production environments."

Reply:
Would be better to not repeat environments. "There are three different environments the application is setup to run in: development, testing, and production."